packages/react/src/components/header/components/headerActionBar/HeaderActionBar.tsx

```diff
@@ -155,7 +155,10 @@ export const HeaderActionBar = ({
         </div>
       </div>
       <HeaderLanguageSelector fullWidthForMobile />
-      <HeaderActionBarNavigationMenu />
+      <HeaderActionBarNavigationMenu
+        logo={logoProps?.href ? <span className={styles.logoWrapper}>{logo}</span> : logo}
+        logoProps={logoProps}
```

Comment:
What if the <span> would always be there? Would do no harm, code would be simpler and possible css quirks easier to fix later when elements do not change via props.

Reply:
https://github.com/City-of-Helsinki/helsinki-design-system/pull/1122/commits/e7fb2d7cd52948d300e9a1548eb620dae7160b1c